cpu/cpu_linux.go

```diff
@@ -37,6 +37,8 @@ func Times(percpu bool) ([]TimesStat, error) {
 func TimesWithContext(ctx context.Context, percpu bool) ([]TimesStat, error) {
 	filename := common.HostProc("stat")
 	var lines = []string{}
+	cpuTotal, _ := common.ReadLinesOffsetN(filename, 0, 1)
+	lines = append(lines, cpuTotal[0])
```

Comment:
I don't think that these two lines are necessary, removing the slice when iterating on `statslines` should be enough IMO